app/config/ServiceFilters.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package config.filters
+package config
```

Comment:
Please could you put the various files you changed back into the `config.filters` package